src/CoreMidi/MidiServices.cs

```diff
@@ -1838,6 +1887,65 @@ internal MidiDevice (MidiDeviceRef handle) : base (handle)
 		internal MidiDevice (MidiDeviceRef handle, bool owns) : base (handle, owns)
 		{
 		}
+#endif // !COREBUILD
+	}
+
+	public class MidiDeviceList : MidiObject {
```

Comment:
@mandel-macaque: How can customers use this class? There are no static members, and no exposed constructors. The class isn't used anywhere else in the PR either.